treeherder/seta/job_priorities.py

```diff
@@ -87,7 +55,7 @@ def _gecko_decision_task_request(project):
             if task_request.counter % jp.priority != 0:
                 job_priorities.append(jp)
 
-    return _process(build_system='taskcluster', job_priorities=job_priorities)
+    return _process(project, build_system='taskcluster', job_priorities=job_priorities)
```

Comment:
Could you add a test that checks passing a non-default project name works? We must be missing coverage.